app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    if command_list[1] == command_list[2]:
+        return
+    if command_list == "cp" and len(command_list) == 3:
```

Comment:
command_list is a list and can't be compared to string in this case. Please, fix it. You should check the zero element of list. 
Also, the check should be performed before checking files identity as @vasylhnatiuk previously mentioned